### PR TITLE
[Stager] Copy Stager from bryanoltman/stager

### PR DESCRIPTION
**Pull request by bryanoltman**
Initial copy of Stager from https://github.com/bryanoltman/stager.

### CODE REVIEW

File: packages/stager/lib/src/scene.dart
Comment (yjbanov) [inline packages/stager/lib/src/scene.dart:61]: This clashes with a core Flutter class: https://api.flutter.dev/flutter/dart-ui/Scene-class.html

Comment (bryanoltman) [reply packages/stager/lib/src/scene.dart:61]: ☹️ 

Is the only solution to rename? I imagine that a file using Stager would probably not also be using dart:ui, but I suppose there's probably someone out there who would want that. Is it awful to suggest using `import as` to disambiguate? i.e., `import 'package:stager/stager.dart' as stager;` and `class MyScene extends stager.Scene` (or `import 'dart:ui' as ui;` and `ui.Scene`)?

Comment (bryanoltman) [reply packages/stager/lib/src/scene.dart]: Renamed to `StagerScene`

Comment (xster) [reply packages/stager/lib/src/scene.dart]: same question. Are people really going to have these imports (configuration of an app-under-test and low level dart:ui access) in the same file? The framework itself already conflicts with dart:ui classes a bunch already and import 'dart:ui' as ui is already a common prior art.

File: packages/stager/example/lib/main.dart
Comment (stuartmorgan) [inline packages/stager/example/lib/main.dart:1]: This file is currently your example tab on pub.dev, which is pretty confusing. You should strongly consider adding an example.md per https://dart.dev/tools/pub/package-layout#examples.

File: packages/stager/pubspec.yaml
Comment (stuartmorgan) [inline packages/stager/pubspec.yaml:21]: This can be removed.

**Review by stuartmorgan — CHANGES_REQUESTED** [review]
I didn't review the code, but just minor nits on the metadata. You'll also need to update
- /README.md to add this package to the table.
- /.github/labeler.yml to add this package to the auto-labeler rules.

Optional since we already don't have it, but would be helpful: I've been planning to add CODEOWNERS to this repository to start clearly mapping packages to their owners. It would be great if you could start it with this package. See [here](https://github.com/flutter/plugins/blob/main/CODEOWNERS) for an example; for now it would just be a single entry with this package it in.

Lastly, you'll need to add me an an uploaded to the package on pub. I need to transfer this to the publisher before this lands, or auto-publish will fail.

**Comment by stuartmorgan** [comment]
Marking as a draft for now, just to get this off of the review queue. Please update the state once it's ready for another round of review!

**Comment by bryanoltman** [comment]
@stuartmorgan I may at some point, although the source package has diverged enough that I will probably just open a new PR.

**Pull request closed without merging**
